system/doc/design_principles/statem.xml

```diff
@@ -1839,12 +1839,12 @@ do_unlock() ->
     io:format("Open~n", []).
     ]]></code>
     <p>
-      The selective receive in this case causes implicitly <c>open</c>
-      to postpone any events to the <c>locked</c> state.
+      The selective receive in this case causes <c>open</c>
+      to implicitly postpone any events to the <c>locked</c> state.
     </p>
     <p>
       A selective receive cannot be used from a <c>gen_statem</c>
-      behavior as for any <c>gen_*</c> behavior,
+      behaviour (or from any <c>gen_*</c> behaviour),
```

Comment:
Could you please change all 8 occurences of "behavior" in the file to "behaviour"?

This is an old internal conflict.  Erlang was created by British guys and so defined the behaviour compiler directive to be `-behaivour`.  Later Ericsson decided to use American English as corporate language, so for example documentation should be written in American English.  Due to that a compiler directive `-behavior` was added as an alias...

Nevertheless.  All documentation in Design Principles use *behaviour* when speaking of an Erlang Behaviour, except these 8 occurrences in `statem.xml`, so you may change them in the name of consistency.

Reply:
> Later Ericsson decided to use American English as corporate language, so for example documentation should be written in American English.

Thanks for this, didn't know that. Will do!

Reply:
So.  American English is the norm.  But when writing about an Erlang Behaviour it should have British spelling.